content/blog/when-to-use-azure-cosmos-db/index.md

```diff
@@ -0,0 +1,290 @@
+---
+title: "When to Use Cosmos DB"
```

Comment:
Make sure to crosslink to other blogs and what-is pages in this post so we can boost SEO of everything